packages/manager/modules/observability/src/components/obsToolbarActions/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { OdsButton } from '@ovhcloud/ods-components/react';
+import {
+  ODS_BUTTON_SIZE,
+  ODS_BUTTON_VARIANT,
+  ODS_BUTTON_ICON_ALIGNMENT,
+  ODS_ICON_NAME,
+  OdsIconName,
+} from '@ovhcloud/ods-components';
+
+export type ObsToolbarAction = {
+  id: string;
+  label: string;
+  link: string;
+  isExternal?: boolean;
+  variant?: ODS_BUTTON_VARIANT;
+  isDisabled?: boolean;
+  icon?: OdsIconName;
+};
+export interface ObsToolbarActionsProps {
+  items: ObsToolbarAction[];
+}
+
+export const ObsToolbarActions = ({
+  items,
+}: Readonly<ObsToolbarActionsProps>): JSX.Element => {
+  const navigate = useNavigate();
+
+  const onActionClick = (link: string, isExternal: boolean) => {
+    if (isExternal) {
+      window.open(link, '_blank');
+    } else {
+      navigate(link);
+    }
+  };
```

Comment:
I purpose to external it (If you want keep).

With a :

```
  const useActionClick = () => {
  const navigate = useNavigate();

return (link: string, isExternal: boolean) => {
    if (isExternal) {
      window.open(link, '_blank');
    } else {
      navigate(link);
    }
  };
}
```

It's easiers to maintain, to test it, and allow reuse